Documentation/MenuObjects/Tmenu/Index.rst

```diff
@@ -9,11 +9,235 @@ TMENU
+
+         The simplest menu TYPO3 can generate is then:
+
+         .. code-block:: typoscript
+            :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+            page.20 = HMENU
+            page.20.1 = TMENU
+            page.20.1.NO = 1
+
+         That will be pure <a> tags wrapped around page titles.
+
+IFSUB
+-----
+
+.. container:: table-row
+
+   Property
+         IFSUB
+
+   Data type
+         boolean / (config)
+
+   Default
+         0
+
+   Description
+         Enable/Configuration for menu items which has subpages.
+
+ACT
+---
+
+.. container:: table-row
+
+   Property
+         ACT
+
+   Data type
+         boolean / (config)
+
+   Default
+         0
+
+   Description
+         Enable/Configuration for menu items which are found in the rootLine.
+
+ACTIFSUB
+--------
+
+.. container:: table-row
+
+   Property
+         ACTIFSUB
+
+   Data type
+         boolean / (config)
+
+   Default
+         0
+
+   Description
+         Enable/Configuration for menu items which are found in the rootLine
+         and have subpages.
+
+CUR
+---
+
+.. container:: table-row
+
+   Property
+         CUR
+
+   Data type
+         boolean / (config)
+
+   Default
+         0
+
+   Description
+         Enable/Configuration for a menu item if the item is the current page.
+
+CURIFSUB
+--------
+
+.. container:: table-row
+
+   Property
+         CURIFSUB
+
+   Data type
+         boolean / (config)
+
+   Default
+         0
+
+   Description
+         Enable/Configuration for a menu item if the item is the current page
+         and has subpages.
+
+USR
+---
+
+.. container:: table-row
+
+   Property
+         USR
+
+   Data type
+         boolean / (config)
+
+   Default
+         0
+
+   Description
+         Enable/Configuration for menu items which are access restricted pages
+         that a user has access to.
+
+SPC
+---
+
+.. container:: table-row
+
+   Property
+         SPC
+
+   Data type
+         boolean / (config)
+
+   Default
+         0
+
+   Description
+         Enable/Configuration for 'Spacer' pages.
+
+         Spacers are pages of the doktype "Spacer". These are not viewable
+         pages but "placeholders" which can be used to divide menu items.
+
+USERDEF1
+--------
+
+.. container:: table-row
+
+   Property
+         USERDEF1
+
+   Data type
+         boolean / (config)
+
+   Description
+         Userdefined, see .itemArrayProcFunc for details on how to use this.
+
+         You can set the ITEM\_STATE values USERDEF1 and USERDEF2 (+...RO) from
+         a script/user function processing the menu item array. See the property
+         .itemArrayProcFunc of the menu objects.
+
+USERDEF2
+--------
+
+.. container:: table-row
+
+   Property
+         USERDEF2
+
+   Data type
+         boolean / (config)
+
+   Description
+         (See above)
+
+
+.. ###### END~OF~TABLE ######
+
+[tsref:(cObject).HMENU.(mObj\_itemStates)]
+
+Order of priority: USERDEF2, USERDEF1, SPC, USR, CURIFSUB, CUR,
```

Comment:
I would like to see this as a note, so it is more prominent (either here or at the beginning of the item states list).